sqliterepo/election.c

```diff
@@ -1636,6 +1636,7 @@ static void
 completion_LISTING(int zrc, const struct String_vector *sv, const void *d)
 {
 	if (!d) return;
+	if (!sv) return;
```

Comment:
This would lead to an election lost in its `LISTING` state. It is necessary that completion hooks make the FSM move, even if it is toward the FAILED state.